src/in_game/in_game.html

```diff
@@ -2,52 +2,55 @@
 <html lang="en">
 <head>
   <meta charset="UTF-8" />
-  <link rel="stylesheet" href="../../css/general.css" />
-  <link rel="stylesheet" href="../../css/header.css" />
-  <link rel="stylesheet" href="../../css/ingame.css" />
-  <link rel="stylesheet" href="../../css/modal.css" />
   <title>InsightEdge - In-Game page</title>
+  <link rel="stylesheet" href="../../css/modifiedingame.css"/>
+<!--  <link rel="stylesheet" href="../../css/general.css" />-->
```

Comment:
blocking: remove comments, no code comments in any of the code.

Reply:
removed